vendor/code.gitea.io/git/scrub.go

```diff
@@ -0,0 +1,32 @@
+package git
+
+import (
+	"os"
+	"path"
+	"fmt"
+	"os/exec"
+)
+
+// Scrub file completely removes a file from a repository's history
```

Comment:
For the sake of consistency, this comment should start with `ScrubFile` (https://blog.golang.org/godoc-documenting-go-code)